geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy

```diff
@@ -189,7 +189,7 @@ IProcess identifyLczType() {
                                         AS SELECT   $ID_FIELD_RSU,
                                                 CASE WHEN IMPERVIOUS_FRACTION_LCZ>PERVIOUS_FRACTION_LCZ AND IMPERVIOUS_FRACTION_LCZ>0.05
                                                         THEN 105
-                                                        ELSE CASE WHEN ALL_VEGETATION<WATER_FRACTION_LCZ OR WATER_FRACTION_LCZ > 0.5
+                                                        ELSE CASE WHEN (ALL_VEGETATION<WATER_FRACTION_LCZ AND WATER_FRACTION_LCZ> 0.5)  OR WATER_FRACTION_LCZ > 0.5
```

Comment:
I think that "OR WATER_FRACTION_LCZ > 0.5" is not useful because redundant and remove the condition "ALL_VEGETATION<WATER_FRACTION_LCZ".
Moreover I am actually not sure about the adding of WATER_FRACTION_LCZ>0.5 because you may have:
building_fraction=0.09
Impervious=0.3
all_vegetation=0.3
water_fraction_lcz=0.31
The sum of all fractions is still 1 but the major part is water. So maybe >0.31 would make more sense ?

Reply:
It's not redundant because you parenthesis precedence.

 The sum of all fractions is still 1 but the major part is water. So maybe >0.31 would make more sense ?
Yes